MAINTAINERS.md

```diff
@@ -0,0 +1,15 @@
+## Overview
+
+This document contains a list of maintainers in this repo. See [RESPONSIBILITIES.md](./RESPONSIBILITIES.md) 
+that explains what the role of maintainer means, what maintainers do in this and other repos, and how they should be 
+doing it. If you're interested in contributing, and becoming a maintainer, see [CONTRIBUTING](CONTRIBUTING.md).
```

Comment:
If I was new to the project it would not be clear to me how to become a maintainer